### PR TITLE
Ecomm 8 order service  repository

### PR DESCRIPTION
**Pull request by BrianVega**
This PR is looking to merge the order service and repository implementation, as well as the orders producers and consumers. The following image shows the output of the main method execution.
image

### CODE REVIEW

File: src/main/java/org/ecommerce/message/broker/consumers/Consumer.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package org.ecommerce.message.broker.consumers;
+
+import org.ecommerce.logs.Log;
+import org.ecommerce.message.broker.MessageQueue;
+import org.ecommerce.models.Product;
+
+public abstract class Consumer<T> extends Log implements Runnable {
```
Comment (algutierrez-griddynamics) [inline src/main/java/org/ecommerce/message/broker/consumers/Consumer.java:7]: This class shouldn't extend from `Log`.
If we want to make use of logging features, we can instantiate a `Log` object instead.

File: src/main/java/org/ecommerce/models/Response.java [hunk 2]
```diff
@@ -24,11 +24,11 @@ public Response(boolean success, String message, HttpStatusCode statusCode) {
 
     @Override
     public String toString() {
-        return "ApiResponse{" +
-                "success=" + success +
-                ", message='" + message + '\'' +
-                ", data=" + data +
-                " statusCode=" + statusCode +
-                '}';
+        return "ApiResponse {" +
```
Comment (algutierrez-griddynamics) [inline src/main/java/org/ecommerce/models/Response.java:27]: Please try the lombok's `@ToString` annotation

File: src/main/java/org/ecommerce/message/broker/consumers/Consumer.java (hunk 1 above)
Comment (BrianVega) [reply src/main/java/org/ecommerce/message/broker/consumers/Consumer.java]: Ok, that was actually something i was wondering, here is the commit [911f6f6d51f1fe666d518fb616a3f904c7569a77](https://github.com/algutierrez-griddynamics/e-commerce/pull/26/commits/911f6f6d51f1fe666d518fb616a3f904c7569a77)

File: src/main/java/org/ecommerce/models/Response.java (hunk 2 above)
Comment (BrianVega) [reply src/main/java/org/ecommerce/models/Response.java]: you're right, here's the commit https://github.com/algutierrez-griddynamics/e-commerce/pull/26/commits/ee36ea6e7896e19a17047c948f2395cfc7d5247f

**Review by algutierrez-griddynamics — APPROVED** [review]

**Pull request merged**